pages/rdm_cycle/processing.md

```diff
@@ -1,36 +1,36 @@
+ * Making changes to data formats such that different data sets will be compatible for integration with each other
  * Changing coding systems or ontologies for the data to bring everything to the same level
- * Filtering data so that only data that is suitable for the project remains.
+ * Filtering data such that only data suitable for the project is retained.
 
- After data processing, data is readied for analysis, and should therefore be available to the members of the project team that need to perform that step.
+ After data processing, clean data is ready for analysis and should therefore be available to the members of the project team that need to perform the next steps.
 
 ## Why is Data Processing important?
 
-Data processing is important to ensure the good quality of the collected data and to prepare for the appropriate data analysis. An accurate documentation of every step done during data processing is key for the reproducibility of your result.
+Data processing is important to ensure good quality of the collected data and to prepare it for meaningful data analysis. Accurate data processing is also essential for combining two or more data sets into a single data set. An accurate documentation of every step done during data processing is key for the reproducibility of your result. Processing data correctly makes it easy to arrange, analyse and also saves a lot of space.
 
 ## What should be considered for Data Processing
 
 The following considerations are important for data processing:
 
-* Does data need to be pseudonymized / anonymized? Not only by removing directly identifying data, but also e.g. names are sometimes visible in images.
-* Are there standards for the encoding of different data fields?
-* Is the data properly documented?
-    * Are encoding choices documented?
-    * Is there a special significance to empty or otherwise special data fields? And is that made explicit?
-    * Are all relationships between data fields explicit (e.g. if a dataset contains "medication" and "disease", is that medication actually used to treat the disease? Or is it e.g. medication that the patient is using for other reasons?)
+* Sensitive data should be Pseudonymised/anonymised. Not only should you remove the directly identifying data, but also be attentive to other sources e.g. names written on images.
+* Appropriate standards for encoding different data fields should be used
+* All steps of anonymisation and encoding should be properly documented. For e.g.
+    * The chosen encoding formats used for data fieds should be documented
+    * The special significance of empty or otherwise special data fields
+    * All relationships between data fields should be made explicit (e.g. if a dataset contains "medication" and "disease", is that medication actually used to treat the disease? Or is it a medication that the patient is using for other reasons?)
 
 ## Where can training materials and events about Data Processing be found?
```

Comment:
Training paragraph should be placed after "related problems", at the bottom of the page.